tests/devices/test_default_qubit_torch.py

```diff
@@ -67,36 +75,35 @@
 
 np.random.seed(42)
 
-
 #####################################################
 # Test matrices
 #####################################################
 
-U = torch.tensor(
+U = np.array(
```

Comment:
Why? This is torch tests, so shouldn't all the variables be in torch?

Reply:
We do arithmetics and convert to a torch tensor within the test case.